Sender/generateTimestamps.cpp

```diff
@@ -0,0 +1,46 @@
+#include "generateTimestamps.h"
+
+RandomGenerator::RandomGenerator(double min, double max, unsigned long seed) :
+        _rand_seed(seed), _generator(seed), _min(min), _max(max), _distribution((min + max) / 2, (max - min) / 4) {}
```

Comment:
for readability you can have each member variable in a seperate line.

Reply:
done.